content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -144,67 +168,77 @@ While it requires some effort, the end result is well worth it
         kind: compute#backendServiceGroupHealth
      ```
 
-    * Both backends should be reported as healthy
-
-    * It can take several minutes for the load balancer to consider the backend healthy
+    Both back ends should be reported as healthy.
+    It can take several minutes for the load balancer to consider the back ends 
+    healthy.
 
-    * The service with port **${NODE_PORT}** is the one we care about most since that is the one handling Kubeflow traffic
+    The service with port `${NODE_PORT}` is the one that handles Kubeflow 
+    traffic.
 
-    * If the backend is unhealthy check the status of the envoy podss
+    If a back end is unhealthy check the status of the Envoy pods:
 
-      ```
-      kubectl -n kubeflow get pods -l service=envoy
-      NAME                     READY     STATUS    RESTARTS   AGE
-      envoy-69bf97959c-29dnw   2/2       Running   2          1d
-      envoy-69bf97959c-5w5rl   2/2       Running   3          1d
-      envoy-69bf97959c-9cjtg   2/2       Running   3          1d
-      ```
-
-      	* The backends should have status Running
+    ```
+    kubectl -n kubeflow get pods -l service=envoy
+    NAME                     READY     STATUS    RESTARTS   AGE
+    envoy-69bf97959c-29dnw   2/2       Running   2          1d
+    envoy-69bf97959c-5w5rl   2/2       Running   3          1d
+    envoy-69bf97959c-9cjtg   2/2       Running   3          1d
+    ```
 
-      	* A small number of restarts is expected since the envoy containers need to be restarted as part of their configuration process
+    * The back ends should have status `Running`.
 
+    * A small number of restarts is expected since the configuration process
+      restarts the Envoy containers.
 
-     * If the pods are crash looping look at the logs to try to figure out why
+    * If the pods are crash looping look at the logs to try to figure out why.
 
-       ```
-       kubectl -n kubeflow logs ${POD}
-       ```
+        ```
+        kubectl -n kubeflow logs ${POD}
+        ```
 
-1. Now that the certificate exists the ingress should report that it is serving on https as well
+        Refer to the [troubleshooting 
+        guide](/docs/gke/troubleshooting-gke/#envoy-pods-crash-looping-root-cause-is-backend-quota-exceeded)
+        for common problems, including exceeded quota.
 
-       ```
-       kubectl -n kubeflow get ingress
-       NAME            HOSTS                                                        ADDRESS          PORTS     AGE
-       envoy-ingress   mykubeflow.endpoints.myproject.cloud.goog   35.244.132.159   80, 443   1d
-       ```
+1. Now that the certificate exists, the Ingress resource should report that it 
```

Comment:
Is "Ingress" the name of the resource? Not sure if this needs to be capitalized or not.

Reply:
I checked the Kubernetes docs and as far as I could tell they capitalize it.